src/program_generator/pg_arguments.py

```diff
@@ -108,6 +108,23 @@ def _do_generate(self, opcode, count, op_count):
 
   def _generate_program_triplet(self, operation, op_count):
     opcode = operation['Mnemonic']
+
+    # the program triplet will be for the following number of measured OPCODEs
+    op_counts = [0, op_count, op_count * 2]
+
+    if opcode == "CREATE":
```

Comment:
To have it generated by default we need to add CREATE opcode to `selection_arguments.csv`

Reply:
This is for another PR. Maybe we should split it into several files and coordinate with marginal program generation. Not sure yet.